scripts/update-substrate-links.mjs

```diff
@@ -10,30 +15,28 @@ import fs from "fs";
   });
   let latestTag = releases.data[0].tag_name;
   let pastTag = releases.data[1].tag_name;
-  let pass = false;
-  while (!pass) {
-    let testUrl = `http://substrate.dev/rustdocs/${latestTag}/sc_service/index.html`;
-    try {
-      const res = await axios.get(testUrl);
-      if (res.status === 200) pass = true;
-    } catch (err) {
-      latestTag = latestTag.slice(0, -1);
-    }
-  }
 
-  let passTwo = false;
-  while (!passTwo) {
-    let testUrl = `http://substrate.dev/rustdocs/${pastTag}/sc_service/index.html`;
-    try {
-      const res = await axios.get(testUrl);
-      if (res.status === 200) passTwo = true;
-    } catch (err) {
-      pastTag = pastTag.slice(0, -1);
+  const testSubstrateDocsVersion = async (tag) => {
+    let tagTemp = tag;
+    let pass = false;
+    
+    while (!pass) {
+      let testUrl = `http://substrate.dev/rustdocs/${tagTemp}/sc_service/index.html`;
+      try {
+        const res = await axios.get(testUrl);
+        if (res.status === 200) pass = true;
+      } catch (err) {
+        tagTemp = tagTemp.slice(0, -1);
+        if (tagTemp.length === 0) {
+          console.log(`NOTE: '${tag}' and variants of its prefixes are not valid Substrate rustdoc version numbers. No links to update!`);
```

Comment:
We can log the err as well

Reply:
I don't think it's noteworthy, every error is always a 404 and will pollute the logs with every prefix try. If we hit that console.log, it means we've tried every prefix